docsite/docs/Reference/variables-and-functions.md

```diff
@@ -82,6 +82,20 @@ As you can see, the query strings support jinja.
 
 Note that the use of `ref` inside `execute_sql` does not create a node in a dbt dag. So in the case of Python models, you still need to specify dependencies in a comment at the top of the file. For more details, [see here](../Docs/fal-cli/python-models.md#dependencies-on-other-models).
 
+### `list_models` function
```

Comment:
want to inlcude `list_sources`, `list_tests` in new PRs?

Reply:
I can do them in one PR.